app/assets/stylesheets/application.css.scss

```diff
@@ -62,11 +62,11 @@ input {
   border: 0px;
 }
 
-.navbar-form > input[type="text"] {
-  width: 200px;
-  height: 30px;
-  border-radius: 4px;
-  border: 0px;
+.navbar-form > form > input[type="text"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3